modules/etcd-creating-automated-backups.adoc

```diff
@@ -0,0 +1,306 @@
+====
+
+:FeatureName: Automating etcd backups
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have access to the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `FeatureGate` custom resource (CR) file named `etcd-enable-automated-backups.yaml` with the following contents:
++
+[IMPORTANT]
+====
+QUESTION: Are there any side effects of the `CustomNoUpgrade` feature set to warn the user about? 
+====
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  customNoUpgrade:
+    enabled:
+      - AutomatedEtcdBackup
+  featureSet: CustomNoUpgrade
+----
+
+. Run the following command to apply the CR and enable automated backups:
++
+[source,terminal]
+----
+$ oc apply -f etcd-enable-automated-backups.yaml
+----
+
+. If dynamically-provisioned storage is available, complete the following steps:
+
+.. Create a persistent volume claim (PVC) named `etcd-backup-pvc.yaml` with contents such as the following example:
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: etcd-backup-pvc
+  namespace: openshift-etcd
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 200Gi <.>
+  storageClassName: standard-csi <.>
+  volumeMode: Filesystem
+----
+<.> The amount of storage available to the PVC. Adjust this value for your requirements.
+<.> The name of the `StorageClass` required by the claim. Adjust this value according to your environment.
++
+[NOTE]
+====
+Each of the following providers requires `ReadWriteMany` for the `accessModes` key:
+
+* Google Cloud Platform
+* Microsoft Azure
+* AWS when using the `versioned-installer-efc_operator-ci` profile
+
+Each listed provider requires the following `storageClassName`:
+
+AWS:: `efs-sc`
+Google Cloud Platform:: `filestore-csi`
+Microsoft Azure:: `azurefile-csi`
+====
+
+.. Apply the PVC by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f etcd-backup-pvc.yaml
+----
+
+.. Verify the creation of the PVC by running the following command:
++
+[source,terminal]
+----
+$ oc get pvc
+----
++
+.Example output
+[source,terminal]
+----
+NAME              STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+etcd-backup-pvc   Pending                                      standard-csi   51s
+----
+
+. If dynamically-provisioned storage is unavailable, create a local storage PVC by completing the following steps:
```

Comment:
a dynamic PVC will always stay Pending until it's mounted, not sure this will confuse anyone

Reply:
Noted, I'll add an admonition to notify the reader.